backend/FwLite/MiniLcm/SyncHelpers/WritingSystemSync.cs

```diff
@@ -0,0 +1,59 @@
+using MiniLcm;
+using MiniLcm.Models;
+using MiniLcm.SyncHelpers;
+using SystemTextJsonPatch;
+
+public static class WritingSystemSync
+{
+    public static async Task<int> Sync(WritingSystem[] currentWritingSystems,
+        WritingSystem[] previousWritingSystems,
+        IMiniLcmApi api)
+    {
+        return await DiffCollection.Diff(api,
+            previousWritingSystems,
+            currentWritingSystems,
+            ws => ws.Id,
```

Comment:
Id will be empty here for FW writing systems making it a poor key choice, we should use `WsId`. We should also use `Type` as those would be separate records in FW and I'm not sure what would happen if you tried to change the type. You can return a tuple here to do that, so it would look like `ws => (ws.WsId, ws.Type)`

Reply:
Changed to a WsId, Type tuple in commit 237b32f1.